Tests/js/views/services/plugins/assets/ez-publishdraftplugin-tests.js

```diff
@@ -595,19 +634,122 @@ YUI.add('ez-publishdraftplugin-tests', function (Y) {
 
             this.view.fire('whatever:publishAction', {
                 formIsValid: true,
-                fields: fields
+                fields: fields,
+                serverSideErrorCallback: serverSideErrorCallback
             });
 
             Assert.isFalse(loading, "The app should not be in loading mode");
             Assert.isTrue(errorNotification, "The notification should have been fired");
         },
     });
 
+    serverSideErrorTest = new Y.Test.Case(Y.merge(Y.eZ.Test.DraftServerSideValidation.CallbackCalledTestCase, {
+        name: "eZ Save Draft Plugin event tests",
+
+        setUp: function () {
+            var that = this;
+
+            this.publishRedirectionUrl = '/something';
+            this.languageCode = 'eng-GB';
+            this.contentId = '/reamonn/supergirl';
+            this.app = new Y.Mock();
+            this.capi = {};
+            this.version = new Y.Mock();
+            this.content = new Y.Mock();
+            this.contentType = {};
+            this.parentLocation = {};
+            this.publishStartedNotification = false;
```

Comment:
this property and a few others seem to be useless. Please keep the setup minimal